terraform/aws/elb/main.tf

```diff
@@ -29,17 +19,33 @@ resource "aws_elb" "elb" {
     interval            = 30
   }
 
+  listener {
+    instance_port     = 443
+    instance_protocol = "tcp"
+    lb_port           = 443
+    lb_protocol       = "tcp"
+  }
+
```

Comment:
how do u propose to get the certificate in there though? is it actually needed? its only needed if you want to terminate SSL at the ELB - my current config just passes it through